src/server/utilities/cspHeader/index.js

```diff
@@ -363,6 +363,7 @@ const helmetCsp = ({ isAmp, isLive }) => ({
     'style-src': generateStyleSrc({ isAmp, isLive }),
     'worker-src': generateWorkerSrc({ isAmp }),
     'report-to': 'default',
+    'upgrade-insecure-requests': true,
```

Comment:
Unfortunately, this directive is not supported by IE11 or Opera Mini. I have tested this with some includes on IE11 via BrowserStack and for some reason, they were behaving correctly as images were loading in with no console errors.

Reply:
The only other way I can think of supporting these browsers is to manipulate them during the render on the server by replacing `http` with `https` before it's served to the user.

Reply:
Ahh forgot about that will double-check with latest. I know that when I was looking around helmet-csp's source code it was doing browser sniffing to apply the different variations of `content-security-policy` to browsers, where IE11 uses `x-content-security-policy`.

Reply:
You are correct @HarveyPeachey, but we don't whitelist the IE headers in mozart so they are never sent to the user on test and live. I imagine though for your browserstack test from local they _were_ present. No big deal, it's nice it works!